processor/configuration/src/main/java/com/redhat/service/smartevents/processor/GatewayConfiguratorService.java

```diff
@@ -10,4 +10,14 @@ public interface GatewayConfiguratorService {
     String getBridgeEndpoint(String bridgeId, String customerId);
 
     String getConnectorTopicName(String processorId);
+
+    String getBootstrapServers();
+
+    String getClientId();
+
+    String getClientSecret();
+
+    String getSecurityProtocol();
+
+    String getBridgeErrorTopicName(String testBridgeId);
```

Comment:
I'm not sure to understand why to expose these in the generic `GatewayConfigurationService` 🤔